website/server/controllers/api-v3/members.js

```diff
@@ -194,12 +270,27 @@ api.getMemberAchievements = {
 };
 
 // copy from rootCtrl => $scope.contribText
-function contribText(contrib, backer, res){
+function contribText (contrib, backer, res) {
   if (!contrib && !backer) return;
   if (backer && backer.npc) return backer.npc;
   var l = contrib && contrib.level;
```

Comment:
Just a nit, but the variable name `l` looks uncomfortably similar to the number `1`, which is confusing. :stuck_out_tongue: